fcp3/node.py

```diff
@@ -3251,8 +3251,8 @@ def uriIsPrivate(uri):
     if "/" in extra:
         extra = extra.split("/")[0]
     extra += "/"
-    extrabytes = base64.decodestring(extra)
-    isprivate = ord(extrabytes[1])
+    extrabytes = base64.decodebytes(extra.encode())
```

Comment:
does this need an explicit `encoding = "utf-8"` argument?

Reply:
According to the help of str.encode 'utf-8' is the default encoding so adding encoding='utf-8' will not add any value.

Another question it if this could have some other encoding. It should be an URI and as such I assume it is limited to ASCII (32 - 126) or perhaps even (33-126) so any utf-coding will probably work just as well.